### ISSUE TITLE
Font Problem

### ISSUE THREAD

**Issue by KitchM** [keepassxreboot/keepassxc#11010]
## Overview
[TIP]:  # ( DO NOT include screenshots of your actual database! )
[NOTE]: # ( Give a BRIEF summary about your problem )
Wrong font is used.

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. Type in a new entry with new password
2. See that the password contains the letter l that looks like the number 1
3.  

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
Must be able to change fonts to one without serifs

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )

## Context
[NOTE]: # ( Give us any additional information you may have. )

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - VERSION 2.7.4
Revision: REVISION 63b2394

[NOTE]: # ( Pick choices based on your environment )
Operating System: Windows/Linux/macOS  Debian 12
Desktop Env: Gnome/KDE/XFCE/Mate/Cinnamon  Xfce 4.18
Windowing System: X11/Wayland X11

**Comment by droidmonkey** [comment]
You are on an outdated version of keepassxc. You also control the fonts for monospace in your operating system settings.

**Comment by KitchM** [comment]
Thanks.  I just changed the default monospace font and I do not see any change in the program.  I may have to log out and log back into the environment to make it work.

By the way, where is the version update option in the program?  I couldn't find it.

**Comment by droidmonkey** [comment]
If you are using the native installation method then you won't see it on Linux. You will have to switch to Flatpak.

**Comment by KitchM** [comment]
On the web site, it appears that appimage is the preferred choice.  I used it and it works.  I got version 2.7.9.  However, when I try to check for updates, I get this response:
"An error occurred when trying to retrieve update information please try again later."

I went to find the Flatpak version you suggested but there are two of them and I cannot figure out the difference.

Thanks.

**Comment by droidmonkey** [comment]
Hmmmm? There is only one flatpak version: https://flathub.org/apps/org.keepassxc.KeePassXC

**Comment by KitchM** [comment]
From the web page:

[Flatpak Package](https://flathub.org/apps/details/org.keepassxc.KeePassXC)

flatpak remote-add --user --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo

flatpak install --user flathub org.keepassxc.KeePassXC

**Comment by droidmonkey** [comment]
First command adds flathub as a repository, second installs keepassxc

**Comment by KitchM** [comment]
I didn't recognize them as commands.  I assumed I was on the download page.

So I ran the commands and I don't have anything on the desktop or menus.  I will log out and back in and see if that helps.

**Comment by KitchM** [comment]
When I logged back in, I found the program listed in the menus under Accessories.  That's good.

However, when I checked the box to make it autostart at boot up, it does not.

Also, there is not, and I cannot add it to the Settings>Session and Startup> Application Autostart because it cannot be seen by the system.

The good news is that I have a better font now where I can at least see a character is a lower case L and not a 1, so there's that.
